src/main/java/com/truelayer/java/paymentsproviders/entities/searchproviders/AuthorizationFlow.java

```diff
@@ -0,0 +1,11 @@
+package com.truelayer.java.paymentsproviders.entities.searchproviders;
+
+import lombok.*;
+
+@Builder
+@Getter
+@ToString
+@EqualsAndHashCode
+public class AuthorizationFlow {
```

Comment:
Needed, right? Like, we don't have already a AuthFlow model with just the config object

Reply:
That's correct, we have another `AuthorizationFlow` entity but it's different (it's a wrapper for an `actions` object and it's used in the start auth flow API response)